workload/docs/baseline-troubleshooting-guide.md

```diff
@@ -10,7 +10,7 @@ Follow the steps below to troubleshoot and resolve the issue:
 
 ### Validate environment and account configuration
 
-- **Check Configuration**: Review your Azure Virtual Desktop (AVD) virtual network configuration and ensure that DNS is properly configured and the virtual network is peered to the network Hub or Identity Services virtual network. When using AD DS or AAD DS commonly the virtual network will need to be setup with custom DNS servers settings that point to the domain controllers IPs.
```

Comment:
AAD DS service hasn't been renamed to the new Microsoft Entra ID model, for now let's keep AAD DS

Reply:
@roarrioj this one seems to be pending for updates.